typescript/packages/well-log-viewer/src/components/WellLogView.tsx

```diff
@@ -827,6 +827,20 @@ function fillPlotTemplate(
     const scale =
         templateTrack?.scale !== options.scale ? options.scale : undefined;
 
+    let colorTableName = "";
+    if (typeof options.colorTable === "function") {
+        colorTableName = "Function";
+    } else if (options.colorTable) {
```

Comment:
replace by:
} else {
colorTableName = options.colorTable?.name ?? "";
}

Reply:
changed it to a conditional (ternary) operator:

```
        colorTable:
            typeof options.colorTable === "function"
                ? "Function"
                : options.colorTable?.name ?? "",
```